tests/bluetooth/controller/ctrl_ull_dletime/src/dle_2m.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018-2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define FEATURES_TO_TEST 4
+#define OCTETS_TO_TEST 5
+
+static u16_t default_octets[] = {27, 60, 128, 200, 251};
+
+static u16_t feature[] = {0,
+		BIT(BT_LE_FEAT_BIT_PHY_2M),
+		BIT(BT_LE_FEAT_BIT_PHY_CODED),
+		BIT(BT_LE_FEAT_BIT_PHY_2M) || BIT(BT_LE_FEAT_BIT_PHY_CODED)};
+static u16_t nr_of_size_elements = ARRAY_SIZE(default_octets);
+#ifdef CONFIG_BT_CTLR_PHY
+static u16_t default_time[] = {328, 2120, 17040};
+static u16_t nr_of_time_elements = ARRAY_SIZE(default_time);
+#endif
+static u16_t nr_of_features = ARRAY_SIZE(feature);
+
+
```

Comment:
Unnecessary empty line

Reply:
sed and awk are nice; extra empty lines removed